presto-docs/src/main/sphinx/release/release-0.226.rst

```diff
@@ -0,0 +1,21 @@
+=============
+Release 0.226
+=============
+
+General Changes
+_______________
+* Add peak running task count to statistics field of QueryCompletedEvent
+* Fix ST_Buffer for tiny geometries (#13194)
+* Fix ST_Centroid for tiny geometries (#10629)
+* Fix bug that causes failure when reading unpartitioned table from some connectors with session property ``grouped_execution_for_eligible_table_scans`` turned on.
+* Reuse compression buffer in PagesSerde when serializing pages.
+* Improve memory tracking for geospatial query indexing and prevent worker node down due to out of memory
+* Use Hilbert Packed RTree (Flatbush) instead of the JTS STR RTree for spatial joins.  The Flatbush is a flattened tree that uses vastly fewer heap allocations, and has faster build and query times.
+
+Hive Changes
+____________
+* Add debug mode that allows reading from offline table and partition. This is controlled by session property ``offline_data_debug_mode_enabled``
+* Fix computation for number of buckets accessed for ``max_buckets_for_grouped_execution``
+* Fix a bug where the bucket column was not available if ``max_buckets_for_grouped_execution`` was exceeded
+* Fix high CPU usage when writing ORC files with too many row groups.
+* Writing into Hive bucketed tables now done in parallel (multiple threads per node) with respect to the task.writer-count configuration property
```

Comment:
- add back ticks around `task.writer-count`
- `Writing into Hive bucketed tables now done in parallel` -> `Increase node-level parallelism of writing into Hive bucketed tables. To enable set ``task.writer-count`` configuration property.`